package.json

```diff
@@ -48,55 +48,55 @@
     "node-graceful": "^2.0.1",
     "node-targz": "^0.2.0",
     "nopt": "3.0.x",
-    "popper.js": "^1.16.0",
-    "profanities": "^2.12.0",
+    "popper.js": "^1.16.1",
+    "profanities": "^2.12.1",
     "promise-queue": "2.2.3",
     "pug": "^2.0.4",
-    "request": "^2.88.0",
+    "request": "^2.88.2",
     "response-time": "^2.3.2",
     "selectize": "0.12.4",
     "semver": "^5.7.1",
     "serve-favicon": "^2.4.5",
-    "shell-quote": "^1.6.3",
+    "shell-quote": "^1.7.2",
     "sinon": "^7.5.0",
     "systemd-socket": "0.0.0",
-    "temp": "0.8.x",
-    "terser-webpack-plugin": "^2.3.4",
+    "temp": "^0.8.4",
+    "terser-webpack-plugin": "^2.3.5",
     "tree-kill": "^1.2.2",
     "underscore": "1.9.1",
     "url-join": "^4.0.1",
     "vis": "4.20.1",
-    "which": "^2.0.1",
+    "which": "^2.0.2",
     "winston": "^3.2.1",
     "winston-papertrail": "^1.0.5",
-    "wolfy87-eventemitter": "^5.2.8"
+    "wolfy87-eventemitter": "^5.2.9"
   },
   "devDependencies": {
-    "@sentry/cli": "^1.49.0",
+    "@sentry/cli": "^1.51.0",
     "aws-sdk-mock": "^1.7.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-http": "^4.2.1",
-    "codecov": "^3.6.1",
+    "codecov": "^3.6.5",
     "copy-webpack-plugin": "^5.1.1",
     "css-loader": "^2.1.0",
     "eslint": "5.16.0",
     "file-loader": "^1.1.5",
     "mini-css-extract-plugin": "^0.7.0",
     "mocha": "^5.2.0",
-    "mock-fs": "^4.10.2",
-    "monaco-editor-webpack-plugin": "^1.7.0",
+    "mock-fs": "^4.10.4",
+    "monaco-editor-webpack-plugin": "^1.9.0",
```

Comment:
hold back monaco-editor-webpack-plugin, since it is tied to the Monaco version itself - this is what caused the Travis build to fail